assets/src/edit-story/panels/scale.js

```diff
@@ -31,30 +32,32 @@ function ScalePanel( { selectedElements, onSetProperties } ) {
 	};
 	return (
 		<Panel onSubmit={ handleSubmit }>
-			<Title>
+			<PanelTitle>
 				{ 'Image actual size' }
-			</Title>
-			<InputGroup
-				label="Scale"
-				value={ typeof state.scale === 'number' ? state.scale : '(auto)' }
-				isMultiple={ scale === '' }
-				onChange={ ( value ) => setState( { ...state, scale: isNaN( value ) || value === '' ? '(auto)' : parseFloat( value ) } ) }
-				postfix="%"
-			/>
-			<InputGroup
-				label="Focal X"
-				value={ typeof state.focalX === 'number' ? state.focalX : '(auto)' }
-				isMultiple={ focalX === '' }
-				onChange={ ( value ) => setState( { ...state, focalX: isNaN( value ) || value === '' ? '(auto)' : parseFloat( value ) } ) }
-				postfix="%"
-			/>
-			<InputGroup
-				label="Focal Y"
-				value={ typeof state.focalY === 'number' ? state.focalY : '(auto)' }
-				isMultiple={ focalY === '' }
-				onChange={ ( value ) => setState( { ...state, focalY: isNaN( value ) || value === '' ? '(auto)' : parseFloat( value ) } ) }
-				postfix="%"
-			/>
+			</PanelTitle>
+			<PanelContent>
+				<InputGroup
+					label="Scale"
+					value={ typeof state.scale === 'number' ? state.scale : '(auto)' }
+					isMultiple={ scale === '' }
+					onChange={ ( value ) => setState( { ...state, scale: isNaN( value ) || value === '' ? '(auto)' : parseFloat( value ) } ) }
```

Comment:
Repeated logic. Make this is a generic arrow function.

Reply:
I didn't write this, just indented the section. Out of scope for this PR.